### DISCUSSION TITLE
Show live-view of pixelit  in home assistant

### DISCUSSION THREAD

**Discussion by Phelton** [pixelit-project/PixelIt#324] [General]
Hi all,
I want find a way for show in the panel of Home Assistant, the live view state of  my pixelit (i want the some that Is possibile view in frontpage web of pixelit)
Something have some idea of how Is possibile realize that? I think that One of possibile way Is grab the live view part from web Page of pixel It...but i'm searching a correct aprotch to this solution or other possibile solution.

Some ideas o suggestion ?

Thanks a lot for your support.

Phelton

**Comment by jekader** [comment]
Liveview is avaiable on the PixelIt in the latest release - would this work for you?

**Comment by Phelton** [comment]
Uhm... I have installed  the version 2.5.0, but don't find a way for share live view in the dashbord of Home Assistant. You have a suggestion ?

Thanks 
Bye

**Reply by foorschtbar** [reply]
Thats not possible at the moment. The liveview a custom protocol who is send over a websocket stream to the webui. For HA somebody needs to write a extra plugin for the HA UI. i cant help with that at the moment :(

**Reply by jekader** [reply]
Sorry, didn't read the question till the end. This is not possible at the moment, however I'd say there is an alternative way of implementing this for use on all sorts of external platforms: generate a BMP file on the fly.

This file format is ancient and extremely simple: a 24-bit BMP would always have the same header and pixel values are just plain RGB:
https://en.wikipedia.org/wiki/BMP_file_format#Example_1

This would still need writing code, but a function similar to `Liveview::fillBuffer()` would suffice to generate the bitmap. It can then be fetched via a static URL by external parties and displayed without much fuss.

**Reply by commputethis** [reply]
I would think that could be possible through Node-Red in Home Assistant.  It shows in node red what is being sent to the display. So, I would think one could come up with a way to have a variable in Home Assistant that changes based on that value.  However, it is just a thought and haven't looked into it at all.  Just seems like the information is there...
